### PR TITLE
1.20.2

### PR DESCRIPTION
**Pull request by mworzala**

### CODE REVIEW

**Comment by MelonHell** [comment]
What is missing to merge this branch?

**Comment by mworzala** [comment]
There are three things I know of:
1. Updating the behavior of `ConnectionManager#getOnlinePlayers`
2. Reentering the configuration phase
3. New chunk batching

I am going to merge before new chunk batching is done, it is not a requirement for the client. On top of those other bits, I would like to try using it a bit in a real world case. I am trying to add branch publishing so that I (and others) can try it before committing to merging it.

**Comment by mworzala** [comment]
Snapshots for this branch (and other PR-ed branches) can be found here: https://repo1.maven.org/maven2/dev/hollowcube/minestom-ce-snapshots/

**Comment by MelonHell** [comment]
I updated my project to Minestom 1.20.2 and it sparkled with new colors :D

image

**Comment by mworzala** [comment]
I have run this on my own server and it appears to be working as expected. I need to do some cleanup in this branch, but it seems functional.

**Comment by MelonHell** [comment]
When a client receives a PLAY packet during CONFIGURATION, the client throws this error. I would really like to have such a check on the server side, and the server would throw an error when trying to send a PLAY packet
image

File: src/main/java/net/minestom/server/entity/metadata/display/AbstractDisplayMeta.java [hunk 1]
```diff
@@ -10,124 +10,132 @@
...
         return super.metadata.getIndex(OFFSET, 0);
     }
 
-    public void setInterpolationStartDelta(int value) {
+    public void setPosRotInterpolationDuration(int value) {
         super.metadata.setIndex(OFFSET, Metadata.VarInt(value));
     }
 
-    public int getInterpolationDuration() {
+    public long getTransformationInterpolationStartDelta() {
```
[the remark this answers is not in the document]
Comment (mworzala) [reply src/main/java/net/minestom/server/entity/metadata/display/AbstractDisplayMeta.java]: No idea, it should be an int.

**Comment by mworzala** [comment]
I'm not sure it's super trivial at the moment, but I'll look into it. I agree the current behavior is pretty bad.

**Review by MelonHell — APPROVED** [review]

**Comment by mworzala** [comment]
This will be superseded by #92. Leaving open for the moment so I can see the diff between the two of them, but future changes should be submitted to that PR instead (this one will be closed later).

**Pull request closed without merging**